src/sentry/static/sentry/app/utils/marked.jsx

```diff
@@ -50,20 +50,21 @@ Renderer.prototype.image = function(href, title, text) {
     out += ' title="' + title + '"';
   }
   out += this.options.xhtml ? '/>' : '>';
-  return dompurify.sanitize(out);
+  return out;
 };
 
 marked.setOptions({
   renderer: new Renderer(),
-  // Disable all HTML input and only accept Markdown
   sanitize: true,
 });
 
 const noParagraphRenderer = new Renderer();
 noParagraphRenderer.paragraph = s => s;
 
 const singleLineRenderer = (text, options) =>
-  marked(text, {...options, renderer: noParagraphRenderer});
+  dompurify.sanitize(marked(text, {...options, renderer: noParagraphRenderer}));
```

Comment:
could probably use `sanitizedMarked` here.

Reply:
Gonna get this merged in for now

Reply:
@billyvg sounds good 👍